### PR TITLE
Mention automatic user creation on Desktop Getting Started page

### PR DESCRIPTION
**Pull request by probakowski**
Currently it's hard to find `Automatic User Creation` docs, we want to make it easier by mentioning it on `Getting started` page.

### CODE REVIEW

File: docs/pages/desktop-access/getting-started.mdx [hunk 1]
```diff
@@ -191,8 +191,9 @@ You can now connect to your Windows desktops from the Teleport Web UI:
 
 ![Connecting to a Windows desktop from the Web UI](../../img/desktop-access/non-ad-connect.png)
 
-## Next Steps
+Teleport can automatically create user on Windows desktop, see the [Automatic User Creation page](./rbac/mdx#automatic-user-creation) to learn more.
```
Comment (probakowski) [inline docs/pages/desktop-access/getting-started.mdx:194]: @gravitational/documentation-team I'm not sure if that's the best place to put it, please modify it as you see fit

Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:194]: I would move this up near line 173 where we discuss `windows_desktop_logins`.

**Review by ptgott — APPROVED** [review]

File: docs/pages/desktop-access/getting-started.mdx [hunk 2]
```diff
@@ -172,6 +172,8 @@ You can restrict access to specific hosts by defining values for
 `windows_desktop_labels`, and adjust the array of usernames this role has access
 to in `windows_desktop_logins`.
 
+Teleport can automatically create users on Windows desktop, see the [Automatic User Creation page](./rbac.mdx#automatic-user-creation) to learn more.
+
 <Admonition type="warning" title="RBAC Configuration">
 Ensure that each Teleport user is only assigned Windows logins that they should
 be allowed to access.
```
[the remark this answers is not in the document]
Comment (probakowski) [reply docs/pages/desktop-access/getting-started.mdx:179]: Not sure about it, it's not a warning, right? It's normal mode of operation

**Review by zmb3 — APPROVED** [review]

**Comment by probakowski** [comment]
Closing, because similar thing was added in https://github.com/gravitational/teleport/pull/31678

**Pull request closed without merging**